UPGRADING.md

```diff
@@ -1,10 +1,64 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.5.4
+
+#### Parameter renaming with :as
+
+Prior to 1.5.4 the [parameter
+renaming](https://github.com/ruby-grape/grape#renaming) with `:as` was directly
+touching the request payload
+([`#params`](https://github.com/ruby-grape/grape#parameters)) while duplicating
+the old and the new key to be both available in the hash. This allowed clients
+to bypass any validation in case they knew the internal name of the parameter.
+Unfortunately, in combination with
+[grape-swagger](https://github.com/ruby-grape/grape-swagger) the internal name
+(name set with `:as`) of the parameters were documented.
+
+From now on the parameter renaming is not anymore done automatically in the
```

Comment:
While it does change behavior, I think the old behavior is broken. Let's say "This behavior was fixed. Parameter renaming is now done when ..."